### DISCUSSION TITLE
uosc添加一个旋转画面的按钮

### DISCUSSION THREAD

**Discussion by zfekin** [hooke007/mpv_PlayKit#208] [Q&A 问答] [answered]
怎么把旋转画面cycle-values video-rotate "90" "180" "270" "0"这个命令添加成按钮，尝试了好多次，请问该怎么实现

**Comment by hooke007** [comment, marked as answer]
```
cycle:{icon}:video-rotate:0/90/180/270[?{tooltip}]
```
或
```
command:{icon}:cycle-values video-rotate 0 90 180 270[?{tooltip}]
```

**Reply by zfekin** [reply]
已实现，非常感谢

**Comment by kli2013** [comment]
怎么能找到icon的名字啊,字体文件里怎么找
还有就是为什么我旋转到180就转不动了  
2种命令都试了

**Reply by zfekin** [reply]
最新版mpv-lazy\portable_config\scripts\uosc\elements\Controls.lua中添加rotate = 'command:rotate_right:cycle-values video-rotate 0 90 180 270?旋转画面'

**Reply by zfekin** [reply]
![图片](https://user-images.githubusercontent.com/110800415/201594121-f39b8eb1-6e05-4260-be1b-e949211c9fa1.png)

**Reply by kli2013** [reply]
谢谢,原来我写的[旋转]这样是错的,不能带括号

**Reply by zhongfly** [reply]
https://fonts.google.com/icons?selected=Material+Icons&icon.style=Rounded
  
这里看图标

**Reply by hooke007** [reply]
这些在脚本说明的帖子都有记录，请仔细阅读
